tunnel.go

```diff
@@ -407,6 +429,33 @@ func (api *API) GetTunnelConfiguration(ctx context.Context, params GetTunnelConf
 	return tunnelDetails, nil
 }
 
+// TunnelConnections gets all connections on a tunnel.
+//
+// API reference: https://api.cloudflare.com/#cloudflare-tunnel-clean-up-cloudflare-tunnel-connections
```

Comment:
this is incorrect; we don't yet have public API documentation for this endpoint. do you have a support ticket i can track this against?

Reply:
Yeah, this endpoint is not documented but as mentioned in the linked issue, it is still getting used by cloudflared, that's how I found the existence of this endpoint. Actually, cloudflared won't be able to list all endpoints if this endpoint doesn't work, so I am inclined to believe that this endpoint is stable, just not documented.

I haven't created a ticket (since I am a free user) so I have created a community [post](https://community.cloudflare.com/t/documentation-unavailable-for-tunnel-connections-endpoint/397023). I am willing to contribute to the documentation too. Let me know what I can do to get this PR merged.

Reply:
i'll poke the team but we really need public documentation here. `cloudflared` is run by the team that owns this service so they can make changes and know the impacts, however, we need to make sure the endpoints in use are also stable for public consumption; not just tools that the team maintain.

Reply:
I understand, the main motivation for this PR is a [tool](https://github.com/beezlabs-org/cloudflare-tunnel-operator) that I am working on which would benefit from having this endpoint in the SDK. Moreover, I am just hoping that this endpoint was mistakenly not documented and hence the associated issue and this PR. I really hope this becomes documented though as this endpoint actually provides useful information. 

Moreover, the URL associated with the API already has a DELETE method associated with it. Having the GET method seems like a no brainer.

Reply:
docs are coming and will land in the next couple of days.